crates/subspace-service/src/mmr/sync.rs

```diff
@@ -163,6 +255,13 @@ where
                 // when we sync the remaining data (consensus and domain chains).
                 if target_position <= starting_position.into() {
```

Comment:
Is it possible for this condition to be true before we send any MMR requests, or do we always want to send at least one?

Reply:
This is only true today, there is no guarantee it will be the case tomorrow, in fact we do want tor Snap sync to be possible from any point in the future. We should have a TODO about this.